py/server/deephaven/time.py

```diff
@@ -70,13 +63,16 @@ def now(system: bool = False, resolution: str = 'ns') -> Instant:
         raise DHError(e) from e
 
 
-def today(tz: TimeZone) -> str:
-    """ Provides the current date string according to the current clock.
+# TODO: rename to system_today or dh_now?  I lean to today.
+# TODO: return string or a python date type?  I lean to string.
```

Comment:
My vote is to remove. Its name clashes with Python built-in `datetime.date.today`.

Reply:
We need to have it available. Without it, replay is not very useful. When replaying, a script needs to be able to get the current date and time to function properly. We can rename or whatever, but I don't think we can remove it.